meta-lxatac-software/files/fs-perms-srv-ro.txt

```diff
@@ -0,0 +1,5 @@
+# This matches the original fs-perms.txt.
+# The only difference is that write permissions for the owner were removed
+# (0755 -> 0555) to prevent anyone from writing to the rootfs /srv instead of
+# the /srv partition.
+/srv				0555	root	root	false - - -
```

Comment:
Perhaps use `0000` to make it clearer that nobody is supposed to use this directory (except as a mount point)?

Reply:
Makes sense. I've updated the PR and also removed a conflict while at it.